### PR TITLE
Grisha videiev

### PR DESCRIPTION
**Pull request by Zickfried**

### CODE REVIEW

File: Zickfried/HW 1/hw1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name = input("What's your name?")
```
Comment (kolyasalubov) [inline Zickfried/HW 1/hw1.py:1]: good job

File: Zickfried/HW 1/hw2.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+a = 5
```
Comment (kolyasalubov) [inline Zickfried/HW 1/hw2.py:1]: good job

File: Zickfried/HW3/hw1.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!""" 
+
+key_words = ["better", "never", "is"]
+word_counts = {}
+
```
Comment (kolyasalubov) [inline Zickfried/HW3/hw1.py:25]: see this implementation:
print("In 'Zen of Python' the word \"better\" repeat", zen_Of_Python.count("better"), 'times')
print("In 'Zen of Python' the word \"never\" repeat", zen_Of_Python.count("never"), 'times')
print("In 'Zen of Python' the word \"is\" repeat", zen_Of_Python.count("is"), 'times')


print(zen_Of_Python.upper()) 



print(zen_Of_Python.replace("i","&"))

File: Zickfried/HW3/hw2.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+number = input("Enter 4 digits number:")
```
Comment (kolyasalubov) [inline Zickfried/HW3/hw2.py:1]: good job

File: Zickfried/HW3/hw3.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+x = 53
```
Comment (kolyasalubov) [inline Zickfried/HW3/hw3.py:1]: good job

File: Zickfried/HW4/hw1.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+num = int(input("Enter your num: "))
```
Comment (kolyasalubov) [inline Zickfried/HW4/hw1.py:1]: good job

File: Zickfried/HW5/hw1.py [hunk 7]
```diff
@@ -0,0 +1,5 @@
+my_list = [int(input("Add your numbers:"))]
```
Comment (kolyasalubov) [inline Zickfried/HW5/hw1.py:1]: good job

File: Zickfried/HW5/hw2.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
+entered_number = int(input("Enter your number to make Fibonacci nummber:"))
```
Comment (kolyasalubov) [inline Zickfried/HW5/hw2.py:1]: good job

File: Zickfried/HW6/hw1.py [hunk 9]
```diff
@@ -0,0 +1,23 @@
+#determining even numbers that are divisible by 2
+first_list = []
```
Comment (kolyasalubov) [inline Zickfried/HW6/hw1.py:2]: look at this implementation:
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')

File: Zickfried/HW6/hw2.py [hunk 10]
```diff
@@ -0,0 +1,10 @@
+my_count = 0
```
Comment (kolyasalubov) [inline Zickfried/HW6/hw2.py:1]: good job

File: Zickfried/HW 1/hw1.py (hunk 1 above)
Comment (Zickfried) [reply Zickfried/HW 1/hw1.py:1]: Thanks

File: Zickfried/HW 1/hw2.py (hunk 2 above)
Comment (Zickfried) [reply Zickfried/HW 1/hw2.py:1]: Thanks

File: Zickfried/HW3/hw2.py (hunk 4 above)
Comment (Zickfried) [reply Zickfried/HW3/hw2.py:1]: Thanks

File: Zickfried/HW3/hw3.py (hunk 5 above)
Comment (Zickfried) [reply Zickfried/HW3/hw3.py:1]: Thanks

File: Zickfried/HW4/hw1.py (hunk 6 above)
Comment (Zickfried) [reply Zickfried/HW4/hw1.py:1]: Thanks

File: Zickfried/HW5/hw1.py (hunk 7 above)
Comment (Zickfried) [reply Zickfried/HW5/hw1.py:1]: Thanks

File: Zickfried/HW6/hw1.py (hunk 9 above)
Comment (Zickfried) [reply Zickfried/HW6/hw1.py:2]: ternary answer, interesting, thanks

File: Zickfried/HW6/hw2.py (hunk 10 above)
Comment (Zickfried) [reply Zickfried/HW6/hw2.py:1]: Thanks

File: Zickfried/HW3/hw1.py (hunk 3 above)
Comment (Zickfried) [reply Zickfried/HW3/hw1.py:25]: I add thish method in my homework, THANKS

Comment (kolyasalubov) [reply Zickfried/HW3/hw1.py:25]: Cool

File: Zickfried/HW7/codewars/Are You Playing Banjo.py [hunk 11]
```diff
@@ -0,0 +1,5 @@
+def are_you_playing_banjo(name):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Are You Playing Banjo.py:1]: good job

File: Zickfried/HW7/codewars/Convert a Number to a String!.py [hunk 12]
```diff
@@ -0,0 +1,2 @@
+def number_to_string(num):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Convert a Number to a String!.py:1]: good job

File: Zickfried/HW7/codewars/Counting sheep.py [hunk 13]
```diff
@@ -0,0 +1,2 @@
+def count_sheeps(sheep):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Counting sheep.py:1]: good job

File: Zickfried/HW7/codewars/Find The Distance Between Two Points.py [hunk 14]
```diff
@@ -0,0 +1,4 @@
+import math
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Find The Distance Between Two Points.py:1]: good job

File: Zickfried/HW7/codewars/Jenny's secret message.py [hunk 15]
```diff
@@ -0,0 +1,4 @@
+def greet(name):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Jenny's secret message.py:1]: good job

File: Zickfried/HW7/codewars/Multiples of 3 or 5.py [hunk 16]
```diff
@@ -0,0 +1,10 @@
+def solution(number):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Multiples of 3 or 5.py:1]: good job

File: Zickfried/HW7/codewars/No yelling!.py [hunk 17]
```diff
@@ -0,0 +1,5 @@
+def filter_words(st):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/No yelling!.py:1]: good job

File: Zickfried/HW7/codewars/Reverse List Order.py [hunk 18]
```diff
@@ -0,0 +1,3 @@
+def reverse_list(l):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Reverse List Order.py:1]: good job

File: Zickfried/HW7/codewars/Reversing Words in a String.py [hunk 19]
```diff
@@ -0,0 +1,6 @@
+def reverse(st):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Reversing Words in a String.py:1]: good job

File: Zickfried/HW7/codewars/Will you make it.py [hunk 20]
```diff
@@ -0,0 +1,5 @@
+def zero_fuel(distance_to_pump, mpg, fuel_left):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/Will you make it.py:1]: good job

File: Zickfried/HW7/codewars/bool_to_word.py [hunk 21]
```diff
@@ -0,0 +1,5 @@
+def bool_to_word(boolean):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/bool_to_word.py:1]: good job

File: Zickfried/HW7/codewars/correct_tail.py [hunk 22]
```diff
@@ -0,0 +1,6 @@
+def correct_tail(body, tail):
```
Comment (kolyasalubov) [inline Zickfried/HW7/codewars/correct_tail.py:1]: good job

File: Zickfried/HW7/hw1.py [hunk 23]
```diff
@@ -0,0 +1,12 @@
+def largest_num(num1, num2):
```
Comment (kolyasalubov) [inline Zickfried/HW7/hw1.py:1]: good job

File: Zickfried/HW7/hw2.py [hunk 24]
```diff
@@ -0,0 +1,21 @@
+def rectandle_area(a, b):
```
Comment (kolyasalubov) [inline Zickfried/HW7/hw2.py:1]: good job

File: Zickfried/HW7/hw3.py [hunk 25]
```diff
@@ -0,0 +1,14 @@
+def chr_in_str(str):
```
Comment (kolyasalubov) [inline Zickfried/HW7/hw3.py:1]: good job

**Pull request merged**